### ISSUE TITLE
Feature request: Flag to show display names instead of accounts on TUI timeline

### ISSUE THREAD

**Issue by hxlnt** [ihabunek/toot#501]
I wonder if there would be any interest in implementing an option to show display names rather than lengthy account names in the `tui` timeline?

I successfully demoed this change locally by editing line 621 of `toot/tui/timeline.py`, and I quite like the results!

```
urwid.Text(("status_list_account", status.original.account), wrap="clip"),
```
to
```
urwid.Text(("status_list_account", status.original.author.display name), wrap="clip"),
```

Screenshot:
![Screenshot from 2024-09-07 13-14-34](https://github.com/user-attachments/assets/267d7e39-eb8c-49e7-b48b-29ed5301cf9c)

**Comment by marathone** [comment]
Looks great

**Comment by ihabunek** [comment]
Added a new option `toot tui --show-display-names`.

You can also make it permanent by adding this to your [settings file](https://toot.bezdomni.net/settings.html).

**~/.config/toot/settings.toml**

```
[commands.tui]
show_display_names = false
```

Released in [0.47.0](https://github.com/ihabunek/toot/releases/tag/0.47.0).
